lighthouse-core/audits/metrics/first-contentful-paint.js

```diff
@@ -27,21 +27,31 @@ class FirstContentfulPaint extends Audit {
       title: str_(i18n.UIStrings.firstContentfulPaintMetric),
       description: str_(UIStrings.description),
       scoreDisplayMode: Audit.SCORING_MODES.NUMERIC,
-      requiredArtifacts: ['traces', 'devtoolsLogs'],
+      requiredArtifacts: ['traces', 'devtoolsLogs', 'HostDevice'],
     };
   }
 
   /**
+   * @param {LH.Artifacts} artifacts
+   * @param {LH.Audit.Context} context
    * @return {LH.Audit.ScoreOptions}
    */
-  static get defaultOptions() {
-    return {
-      // 75th and 95th percentiles HTTPArchive -> median and PODR
-      // https://bigquery.cloud.google.com/table/httparchive:lighthouse.2018_04_01_mobile?pli=1
-      // see https://www.desmos.com/calculator/2t1ugwykrl
-      scorePODR: 2000,
-      scoreMedian: 4000,
-    };
+  static getDefaultScoreOptions(artifacts, context) {
+    if (context.settings.emulatedFormFactor === 'mobile' || artifacts.HostDevice === 'mobile') {
```

Comment:
I think this logic is right. @paulirish can you confirm?

Reply:
Emulating desktop on a mobile device doesn't make much sense to me. Shouldn't those runs be scored with numbers tuned for a mobile device?

Reply:
> Emulating desktop on a mobile device doesn't make much sense to me. Shouldn't those runs be scored with numbers tuned for a mobile device?

it doesn't make much sense why someone would be doing that to me either :) but if lantern doesn't correctly emulate desktop there it should be fixed at that level, not patched up at the scoring level

Reply:
Yah I agree that `TestedAsMobileDevice` gives us the signal we want.  
real mobile hardware + desktop emulation is stupid but that's solved with a warning telling the developer they're stupid rather than mis-scoring them.   

Thank god we don't have first-class support for tablet with a mouse or whatever. :)

----

I still like having HostFormFactor in the artifacts tho.. even if we aren't using it here.
I guess that could be a separate PR, but no big deal.